### DISCUSSION TITLE
It seems that everything in the authentication part and access error part is in the process of change.

### DISCUSSION THREAD

**Discussion by Goodday777** [dsdanielpark/Bard-API#277] [General]
I think it will be difficult to use for a while.

**Comment by dsdanielpark** [comment]
We are debugging from multiple angles, but the current issue lies in the use of various API formats across different countries and accounts. The variations in cookie values and access methods make debugging challenging. Additionally, the limited access to Gemini prevents thorough debugging and updating of the package. At the moment, we are only addressing some issues in the current package and preparing for a clean transition to a new package.
